scripts/helperFunctions/validateAllDirectories/validateLogDirectory.ts

```diff
@@ -0,0 +1,9 @@
+import { constants } from "fs/promises";
+import { validateDirectory } from "./validateDirectory";
+
+
+
+
+export const validateLogDirectory = async (): Promise<void> => {
+    validateDirectory("logs", constants.W_OK);
```

Comment:
`await` missing.

Reply:
We don't actually need an await here, as we aren't consuming the result of the promise in this function. By writing it this way, validateLogDirectory will return a promise that will resolve once the validateDirectory promise resolves. Actually though, having this function be async at all adds a bit of unnecessary overhead, so I changed it to be more efficient and also hopefully more clear.